### PR TITLE
feat: samradsgat project

### PR DESCRIPTION
**Pull request by diljath**
## What

Initial deployment of samradsgatt to dev with infra setup.
Has the beginings of design on frontpage but otherwise an empty project.

## Why

To start devolping samradsgatt

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by eirikurn — COMMENTED** [review]
Initial review of core files and setup. 

Let me know when you feel like this is ready for main. 👍

File: apps/samradsgatt/proxy.config.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "/app/samradsgatt/api/graphql": {
```
Comment (eirikurn) [inline apps/samradsgatt/proxy.config.json:2]: Should copy this file from the web project. It basically maps "/api" to the graphql api url.

That way you can call the API locally like in production where this project is on /samradsgatt and the api is on the same domain /api.

File: apps/samradsgatt/server.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { bootstrap } from '@island.is/infra-next-server'
+import proxyConfig from './proxy.config.json'
+
+bootstrap({
+  name: 'samradsgatt',
+  appDir: 'apps/samradsgatt',
+  proxyConfig,
+  externalEndpointDependencies: process.env.BASE_URL
+    ? [process.env.BASE_URL]
+    : [],
```
Comment (eirikurn) [inline apps/samradsgatt/server.ts:10]: BASE_URL is not defined in the infra file. I don't think we need to do a DNS health check against the deployed BASE_URL (or why auth-admin-web does it). Probably fine to remove.

**Review by eirikurn — APPROVED** [review]
LGTM with minor comments 🙌

File: apps/samradsgatt/infra/samradsgatt.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      },
+      NEXTAUTH_URL: {
+        dev: ref(
+          (ctx) =>
+            `https://${
+              ctx.featureDeploymentName ? `${ctx.featureDeploymentName}-` : ''
+            }loftbru.dev01.devland.is`,
+        ),
+        staging: 'https://loftbru.staging01.devland.is',
+        prod: 'https://loftbru.island.is',
```
Comment (brynjarorng) [inline apps/samradsgatt/infra/samradsgatt.ts:32]: Is this app supposed to reference the loftbru app here?

File: apps/samradsgatt/infra/samradsgatt.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import { ref, service, ServiceBuilder } from '../../../infra/src/dsl/dsl'
+
+export const serviceSetup = (): ServiceBuilder<'samradsgatt'> =>
+  service('samradsgatt')
+    .namespace('samradsgatt')
+    .liveness('/liveness')
+    .readiness('/readiness')
```
Comment (brynjarorng) [inline apps/samradsgatt/infra/samradsgatt.ts:7]: I recommend using /liveness here

File: apps/samradsgatt/infra/samradsgatt.ts (hunk 3 above)
Comment (brynjarorng) [inline apps/samradsgatt/infra/samradsgatt.ts:32]: Is this also supposed to reference loftbru?

File: apps/samradsgatt/infra/samradsgatt.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      max: 30,
+      min: 5,
+    })
+    .resources({
+      limits: { cpu: '400m', memory: '512Mi' },
+      requests: { cpu: '200m', memory: '256Mi' },
+    })
+    .env({
+      BASEPATH: '/samradsgatt',
+      IDENTITY_SERVER_ISSUER_URL: {
```
Comment (brynjarorng) [inline apps/samradsgatt/infra/samradsgatt.ts:19]: Are you using the IDS? I can't see it being used in the code

**Review by brynjarorng — APPROVED** [review]
LGTM(Devops)

**Pull request merged**